nosql/userProfile.json

```diff
@@ -0,0 +1,16 @@
+{
+  "id": "Number",
+  "firstName": "String",
+  "lastName": "String",
+  "headline": "String",
+  "industry": "String",
+  "personalInfo": {"phone" : "String", "email": "String", "adress" : "String", "dob": "String"},
+  "numConnections": "Number",
+  "numFollowers": "Number",
+  "summary": "String",
+  "positions": {"title" : "String", "summary" : "String", "start-date" : "String", "end-date" : "String", "is-current" : "Boolean", "company" : {"id" : "String", "name" : "String"}},
+  "pictureUrl": "String",
+  "cvUrl": "String",
+  "skills": ["String"],
```

Comment:
you can also add Interests and Accomplishments(list of how many projects, certificates achieved)

Reply:
@Nabila63Ahmed The basic requirements in the description are posting CV & Skills, so i think any additional information can be added optionally after we finish testing those commands